### DISCUSSION TITLE
Another Vercel account is using this domain.

### DISCUSSION THREAD

**Discussion by 0xhema** [vercel/community#2626] [Help] [answered]
### Summary

I was having issues with my domain so I deleted it and re-added it to the same deployment in vercel and now I am getting this:

<img width="920" alt="Screenshot 2023-05-26 at 4 50 03 PM" src="https://github.com/vercel/community/assets/26447374/c02e972d-fe73-440c-b58b-d1d8221bc240">

### Steps to Reproduce

I was having issues with my domain so I deleted it and re-added it to the same deployment in vercel and now I am getting this:

**Comment by amyegan** [comment]
Sorry you ran into trouble with your domain, @0xhema. If you add the `TXT` record with your DNS provider, that will let Vercel's systems know the correct project to link with the domain. Once the record is propagated, the domain should work like normal with your project.

**Comment by 0xhema** [comment]
The name servers are no longer with vercel, I removed them since it wouldn't let me change them.  so now the name servers are with aws route 53, i cant connect them to vercel at all

**Comment by 0xhema** [comment]
Vercel still thinks my name servers are with it, how can i fix this? @amyegan

**Comment by amyegan** [comment, marked as answer]
Hi @0xhema. I'm not sure what steps you have already tried, but the documentation on [Adding & Configuring a Custom Domain](https://vercel.com/docs/concepts/projects/domains/add-a-domain#) is a good place to start. You should be able to add DNS records with AWS Route 53 to point the domain to your Vercel project. A `TXT` record may be needed to verify domain ownership.

**Comment by PradyumnTiwari** [comment]
Hello Team,
I am facing the same issue.
The domain was added in my other vercel account and I have remove that domain from that account and trying to add in current account.

But there are no any luck, showing the same error which is below.

"Another Vercel account is using this domain. Please set the following TXT record on _vercel.mydomain.com to use subdomain.mydomain.com in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed."

I am usgin Vercel API to add the subdomain in my project in that case I can not add manually TXT record for the same.
I have already added CNAME with value cname.vercel-dns.com, one of my verified domain is working fine, but I need to access by dynamically added domains.

Please help me to resolve this issue.

Thanks in advance.

**Comment by PradyumnTiwari** [comment]
Hello team,
This issue is fixed now, after removing that domain from my old account from the below section
https://vercel.com/dashboard/domains

Thank you :)
